### PR TITLE
Use meta key + wheel to zoom in/out

### PR DESCRIPTION
**Pull request by nanot1m**
Borrowed the idea from figma (:

### CODE REVIEW

**Comment by enzoferey** [comment]
Nice 👌

File: src/index.tsx [hunk 1]
```diff
@@ -1983,10 +1983,17 @@ export class App extends React.Component<any, AppState> {
     e.preventDefault();
     const { deltaX, deltaY } = e;
 
-    this.setState({
-      scrollX: this.state.scrollX - deltaX / this.state.zoom,
-      scrollY: this.state.scrollY - deltaY / this.state.zoom,
-    });
+    if (e[KEYS.META]) {
+      this.setState(({ zoom }) => ({
+        zoom: Math.max(Math.min(zoom - deltaY / 100, 2), 0.1),
```
Comment (enzoferey) [inline src/index.tsx]: There is a clamp zoom function [here](https://github.com/excalidraw/excalidraw/blob/master/src/actions/actionCanvas.tsx#L59) that we should be using. Probably we want to move it to `src/scene/zoom.ts`.

Comment (nanot1m) [reply src/index.tsx]: Updated, thanks!

**Comment by enzoferey** [comment]
Looks good to me 👍

**Comment by dwelle** [comment]
On my mouse which doesn't have high-precision wheel, the step is `100`.

**Comment by nanot1m** [comment]
@dwelle we shall try this https://www.npmjs.com/package/normalize-wheel to fix wheel issues

**Comment by dwelle** [comment]
That won't help, that's just a cross-browser normalization (I tried it just to be sure). We'll need to multiply the `deltaY` by some factor (for really high precision wheels), and clamp it from bottom and top.

**Comment by dwelle** [comment]
I'll make a PR.

**Pull request merged**